trade/api/stakecube_api.js

```diff
@@ -48,10 +47,9 @@ module.exports = function() {
 
     try {
       if (scStatus) {
-        resolve(scResult);
+        resolve(scData);
       } else if ([200, 201].includes(httpCode) && scData) {
-        scResult.errorMessage = scError;
-        resolve(scResult);
+        resolve(scData);
```

Comment:
Add `scData.errorMessage = errorMessage;`
And `log.log('StakeCube processed a request to ${url} with data ${reqParameters}, but with error: ${errorMessage}. Resolving…');`